README.md

```diff
@@ -134,6 +134,63 @@ s.number.negative; // .lt(0)
 s.number.divisibleBy(5); // TODO | Divisible by 5
 ```
 
+And transformations:
+
+```typescript
+s.number.abs;  // TODO | Transforms the number to an absolute number
+s.number.sign; // TODO | Gets the number's sign
+
+s.number.trunc;  // TODO | Transforms the number to the result of Math.trunc`
+s.number.floor;  // TODO | Transforms the number to the result of Math.floor`
+s.number.fround; // TODO | Transforms the number to the result of Math.fround`
+s.number.round;  // TODO | Transforms the number to the result of Math.round`
+s.number.ceil;   // TODO | Transforms the number to the result of Math.ceil`
+```
+
+#### BigInts
+
+ShapeShift includes a handful of number-specific validations:
+
+```typescript
+s.bigint.gt(5); // > 5
+s.bigint.ge(5); // >= 5
+s.bigint.lt(5); // < 5
+s.bigint.le(5); // <= 5
+s.bigint.eq(5); // === 5
+s.bigint.ne(5); // !== 5
+
+s.bigint.finite;  // value must be finite
+
+s.bigint.positive; // .ge(0)
+s.bigint.negative; // .lt(0)
+
+s.bigint.divisibleBy(5); // TODO | Divisible by 5
+```
+
+And transformations:
+
+```typescript
+s.number.abs;  // TODO | Transforms the bigint to an absolute bigint
+
+s.bigint.intN(5);  // TODO | Clamps to a bigint to a signed bigint with 5 digits, see BigInt.asIntN
+s.bigint.uintN(5); // TODO | Clamps to a bigint to an unsigned bigint with 5 digits, see BigInt.asUintN
```

Comment:
```suggestion
s.bigint.intN(5n);  // TODO | Clamps to a bigint to a signed bigint with 5 digits, see BigInt.asIntN
s.bigint.uintN(5n); // TODO | Clamps to a bigint to an unsigned bigint with 5 digits, see BigInt.asUintN
```

Reply:
Actually, those take numbers, see [`BigInt.asIntN()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/asIntN) and [`BigInt.asUintN()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/asUintN)

Reply:
It shouldn't take number, and I'm dying on that hill. This is a bigint validator, not a numberesque one

Reply:
Still, that method takes a number.